src/pkg/cluster/cluster.go

```diff
@@ -35,9 +42,22 @@ const (
 
 // Cluster Zarf specific cluster management functions.
 type Cluster struct {
-	Clientset  kubernetes.Interface
+	// Clientset implements k8s client api
+	Clientset kubernetes.Interface
+	// RestConfig holds common options for a k8s client
 	RestConfig *rest.Config
-	Watcher    watcher.StatusWatcher
+	// Watcher implements kstatus StatusWatcher
+	Watcher watcher.StatusWatcher
+}
+
+// NewWithWait wraps NewClusterWithWait to provide a shorthand alias.
+func NewWithWait(ctx context.Context) (*Cluster, error) {
+	return NewClusterWithWait(ctx)
+}
+
+// New wraps NewCluster to provide a shorthand alias.
+func New(ctx context.Context) (*Cluster, error) {
+	return NewCluster(ctx)
```

Comment:
While I understand the strategy to avoid breaking changes, I personally don't think we're at the point where we should do this for the cluster package. It's unlikely that more than a few projects consume these functions. Also the parameters of `NewCluster` are changing anyway so the user would still have to update for a breaking change.

Reply:
I agree with that, updated in latest to go with `cluster.New()`